infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -191,7 +191,7 @@ private void initializeMetadata(TxnContext txn, LogReplicationSession session, b
 
                 ReplicationStatus defaultSinkStatus = ReplicationStatus.newBuilder()
                         .setSinkStatus(SinkReplicationStatus.newBuilder()
-                                .setDataConsistent(false)
+                                .setDataConsistent(true)
```

Comment:
Could you please add a comment or description on why this is needed, to help understand the context better?

Does it mean when sink side comes up, by default the data consistent field would be true?